entry_types/scrolled/package/src/frontend/Chapter.js

```diff
@@ -4,7 +4,7 @@ import Section from './Section';
 
 export default function Chapter(props) {
   return (
-    <div id={'chapter' + props.permaId}>
+    <div id={'chapter-'+props.permaId}>
```

Comment:
Just as a aside: A template string, i.e. `` `chapter-${props.permaId}` ``, would be more modern/idiomatic, I think